modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: CONCEPT
+[id="security-context-constraints-psa-synchronization_{context}"]
+= Security context constraint synchronization with pod security standards
+
+{product-title} implements link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Globally, the `privileged` profile is enforced, and the `restricted` profile is used for warnings and audits.
+
+In addition to the global pod security admission control configuration, a pod admission controller applies pod security admission control `warn` and `audit` labels to namespaces according to the SCC permissions of the service accounts that are in a given namespace.
+
+The controller examines `ServiceAccount` object permissions to apply pod security profiles constraints to each namespace. Security context constraints (SCCs) are mapped to pod security profiles based on their field values; the controller uses these translated profiles. Pod security admission `warn` and `audit` labels are set to the most privileged pod security profile found in the namespace to prevent warnings and audit logging as pods are created.
+
+Namespace labeling is based on consideration of namespace-local service account privileges.
+
+Applying pods directly might use the SCC privileges of the user who runs the pod. However, user privileges are not considered during automatic labeling.
+
+[IMPORTANT]
+====
+SCC synchronization is disabled by default for user-created namespaces that are named with the prefix `openshift-`. Namespaces that have this prefix are typically system namespaces; by convention, a controller exists to manage them. 
+
+You can enable SCC synchronization in namespaces that have the prefix `openshift-` by setting the value of the `security.openshift.io/scc.podSecurityLabelSync` label to `false`. 
+
+Namespaces that the installer deploys cannot be opted into SCC synchronization. These namespaces are system namespaces.
```

Comment:
We need to describe exactly what are the namespaces. That is not 100% true ( i.e openshift-operators is created by the installed and is not in this list. ) 
```suggestion
Namespaces which are considered as part of OCP payload do not allow SCC synchronization. You can verify the [list](https://github.com/openshift/cluster-policy-controller/blob/release-4.11/pkg/psalabelsyncer/podsecurity_label_sync_controller.go#L42-L111) of namespaces those is not allowed the synchronization.
```

Reply:
indeed, as I stated in my comments about this being vague on purpose and openshift-operators NS being an exception